### ISSUE TITLE
Servers can be stalled with one remaining spectator after a game has finished.

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#14580]
```
13:17 <+pchote> two US official servers are showing very long games active with zero players and one spectator
13:18 <+pchote> I think that can happen if a spectator AFKs and then the players leave
13:18 <+pchote> the game isn't counted as over until everyone has left
13:20 <+pchote> we could add a ServerTrait that will automatically terminate a game say 5 minutes after someone wins
13:22 < jaZz_KC2> this happends rather often, actually. the spec isnt  even in the server anymore, it sometimes hangs with last spec, although that spec is long gone
13:24 < jaZz_KC2> I noticed that a few months ago, when I asked someone if they had two clients open/hogging one server although they entered another one long ago, Said person said he had only one client open and left that server long ago
13:24 <+pchote> ok well this would be a good thing to fix for the playtest
```

**Comment by pchote** [comment]
I feel like i've already scope creeped the milestone too much, so removing.

**Comment by LipkeGu** [comment]
Servers can also stalled as spectator when starting games with no players.
![neue bitmap](https://user-images.githubusercontent.com/2057932/34551395-c4d8032a-f119-11e7-9dd8-cd78d6663ba4.png)

**Comment by abcdefg30** [comment]
Closing as reverse duplicate of #15454 which has a more actionable title. The problem of people starting a game without any players at all (which means the game will never be "over") is addressed by #20086.
